src/bin/deku_node.ml

```diff
@@ -138,6 +138,21 @@ let handle_receive_user_operation_gossip =
       Flows.received_user_operation (Server.get_state ()) update_state
         request.user_operation)
 
+(* POST /user-operations-gossip *)
+(* Propagate a batch of user operations (core_user.t) over gossip network *)
+let handle_receive_user_operations_gossip =
+  handle_request
+    (module Network.User_operations_gossip)
+    (fun update_state request ->
+      let operations = request.user_operations in
+      Format.eprintf "packed: %d\n%!" (List.length operations);
```

Comment:
Please use `Log.debug`

Reply:
Oh, this is a debug message that should e removed. Good catch.